api-test/Helpers/EditionHelpers.cs

```diff
@@ -18,7 +19,7 @@ public static class EditionHelpers
         /// </summary>
         /// <param name="userId">Id of the user whose editions should be randomly selected.</param>
         /// <param name="jwt">A JWT can be added the request to access private editions.</param>
-        /// <returns></returns>
+        /// <returns>a randomly selected EditionDTO</returns>
```

Comment:
Why do we need a randomly selected edition?

Reply:
I figured that it would keep the integration tests fair by sampling different editions each time.